topic7/src/main/java/controllers/CourseController.java

```diff
@@ -0,0 +1,51 @@
+package controllers;
+
+import models.Course;
+import models.CourseDao;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
+public class CourseController {
+
+	@Autowired
+	private CourseDao _CourseDao;
+
+	@RequestMapping("/create")

```

Comment:
Just to know more about it, please read this post http://www.journaldev.com/3358/spring-mvc-requestmapping-annotation-example-with-controller-methods-headers-params-requestparam-pathvariable